pkgs/development/libraries/libtiff/default.nix

```diff
@@ -17,15 +18,25 @@ stdenv.mkDerivation rec {
     sha256 = "0d46bdvxdiv59lxnb0xz9ywm8arsr6xsapi5s6y6vnys2wjz6aax";
   };
 
-  outputs = [ "bin" "dev" "out" "man" "doc" ];
+  # FreeImage needs this patch
+  patches = [ ./headers.patch ];
 
-  nativeBuildInputs = [ pkgconfig ];
+  outputs = [ "bin" "dev" "dev_private" "out" "man" "doc" ];
```

Comment:
dev_private lacks motivation in code and in the commit message. What is this for? (No I am not reading the whole thread if I need to find this out when seeing a non-standard output in the code)

Reply:
there is a comment just above explaining it

Reply:
Though I will modify the commit message